src/pages/en/guides/markdown-content.md

```diff
@@ -395,9 +424,14 @@ We also suggest [diving into their theme documentation](https://github.com/shiki
 #### Prism configuration
 
 When using Prism, you'll need to add a stylesheet to your project for syntax highlighting. If you're just getting started and prefer to use Prism over Shiki, we suggest:
+
 1. [Setting `syntaxHighlight: 'prism'`](#choose-a-syntax-highlighter) from your `@astrojs/markdown-remark` config.
```

Comment:
I’m noticing that `remark-smartypants` turns `syntaxHighlight: 'prism'` into `syntaxHighlight: ‘prism’`, which isn’t great for copy-pasting. Maybe we want to look at disabling SmartyPants for the docs? I don’t think it adds anything essential or can it be configured to ignore code snippets?

Reply:
I think this will be fixed by #404